azkaban-common/src/main/java/azkaban/server/AzkabanServer.java

```diff
@@ -71,11 +85,41 @@ public static Props loadProps(final String[] args, final OptionParser parser) {
         azkabanSettings = loadAzkabanConfigurationFromDirectory(dir);
       }
     } else {
-      logger
-          .info("Conf parameter not set, attempting to get value from AZKABAN_HOME env.");
+      logger.info("Conf parameter not set, attempting to get value from AZKABAN_HOME env.");
       azkabanSettings = loadConfigurationFromAzkabanHome();
     }
 
+    try {
+      if (options.has(emailDirectory)) {
```

Comment:
There are some code duplication in load conf and load email. Can we consolidate the code?

Reply:
we can; updating...